crates/vault-registry/rpc/runtime-api/src/lib.rs

```diff
@@ -19,10 +19,10 @@ sp_api::decl_runtime_apis! {
 
         /// Get the first available vault with sufficient collateral to fulfil an issue request
         /// with the specified amount of PolkaBTC.
-        fn get_first_vault_with_sufficient_collateral(amount: BalanceWrapper<PolkaBTC>) -> Result<AccountId, DispatchError>;
+        fn get_pseudorandom_vault_with_sufficient_collateral(amount: BalanceWrapper<PolkaBTC>) -> Result<AccountId, DispatchError>;
 
         /// Get the first available vault with sufficient tokens to fulfil a redeem request
-        fn get_first_vault_with_sufficient_tokens(amount: BalanceWrapper<PolkaBTC>) -> Result<AccountId, DispatchError>;
+        fn get_pseudorandom_vault_with_sufficient_tokens(amount: BalanceWrapper<PolkaBTC>) -> Result<AccountId, DispatchError>;
```

Comment:
let's not change the name for the public RPC call in this PR, please. This introduces a breaking change in the lib.